src/EnergyPlus/SolarReflectionManager.cc

```diff
@@ -1109,8 +1109,8 @@ namespace SolarReflectionManager {
         // sky diffuse horizontal irradiance
         bool hitObs; // True iff obstruction is hit
 
-        Real64 const DPhi(DataGlobalConstants::PiOvr2 / (AltAngStepsForSolReflCalc / 2.0));      // Altitude angle and increment (radians)
-        Real64 const DTheta(2.0 * DataGlobalConstants::Pi / (2.0 * AzimAngStepsForSolReflCalc)); // Azimuth increment (radians)
+        Real64 const DPhi(Constant::PiOvr2 / (AltAngStepsForSolReflCalc / 2.0));      // Altitude angle and increment (radians)
+        Real64 const DTheta(2.0 * Constant::Pi / (2.0 * AzimAngStepsForSolReflCalc)); // Azimuth increment (radians)
```

Comment:
I recall a `1.0 * something * 1.0 ...`. Now here is a 2.0 / 2.0, and both of these const D* are constexpr

Reply:
At least there are parentheses!!